test/unit/views/payment-sheet-views/card-view.js

```diff
@@ -192,6 +192,36 @@ describe('CardView', function () {
       expect(this.element.querySelector('[data-braintree-id="postal-code-field-group"]')).not.to.exist;
     });
 
+    it('has cardholderName if provided in merchant configuration', function () {
```

Comment:
should we add tests around whether or not cardholder name is required and hiding/showing errors